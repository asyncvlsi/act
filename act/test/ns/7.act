template <pint x>
defproc foo(bool a[x], b) ;

deftype foo <: int<3> (bool q) ;

