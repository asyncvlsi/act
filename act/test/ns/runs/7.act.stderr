ERROR: File `7.act', line 4, col 8
	Name `foo' already used in a previous type definition
